enhancements/two-node-fencing/tnf.md

```diff
@@ -242,6 +253,12 @@ The plan is for this to be changed by one of the nodes during pacemaker initiali
 While set to `External`, CEO will still need to render the configuration for the etcd container in a place where it can be consumed by pacemaker. This ensures that the etcd instance managed by pacemaker can be updated
 accordingly in the case of a upgrade event or whenever certificates are rotated.
 
+In case CEO will be used as the in-cluster operator responsible for setting up Pacemaker fencing it'll require root permissions which are currently mandatory to run the required pcs commands.
+Some mitigation or alternatives might be:
+- Use a different (new) in-cluster operator to set up Pacemaker fencing
```

Comment:
which will have its own challenges:
- introducing a new core operator will cause discussions, since the general goal is to reduce the OCP release payload
- adding that new operator to the release payload will be more work (release payload changes, CI setup, ...) than just adding an operand to CEO
- the new operator will still need root, which might trigger the same discussions as if we use CEO, maybe just with other folks...

Reply:
No argument, I am just not sure if we want to go into that much details for an alternative mitigation which isn't our preferred path of action WDYT ?

Reply:
My idea was to make more obvious that this isn't an easy or preferable alternative :) 
Up to you if you want to add it.

Reply:
I think you have 2 directions - providing a condense list of reasons we rejected these alternatives OR just omit the alternatives entirely. I think Marc's idea is good because I see a reviewer immediately jumping to the question, so why was this alternative rejected?